lib/membrane_rtc_engine/events/switch_track_variant.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Membrane.RTC.Engine.Event.SwitchTrackVariant do
+  @moduledoc """
+  Event for changing track variant that is currently being received.
+
+  Engine will start sending a new track variant as soon
+  as it receives a keyframe for the new variant.
+  If the requested variant is incorrect, the event will
+  be ignored.
```

Comment:
I think this shouldn't be specified here - it's not part of the event's semantics and seems up to the element handling it to ignore it or not, so if the receiver changes its approach, it might be easy to leave this comment outdated.
Besides, ignoring the invalid value doesn't seem correct to me and if by design such a switch might be rejected, then maybe the name should reflect that more, e.g. `RequestTrackVariant` or similar

Reply:
This is event is going to be handled by RTC Engine itself so it's hard to find better place to include this information. We could also raise but ignoring seems to me to be safer option at the beginning

Reply:
> ignoring seems to me to be safer option at the beginning

I'd say it's just the opposite, as it generates a risk of the error being silently ignored. 'at the beggining' suggests it might be changed in future - but this way it's most likely going to stay forever :stuck_out_tongue: I'd rather `raise` and change the behaviour back to ignore only as a workaround when we encounter a problem we won't be willing to fix.
The above assumes we're talking about a really incorrect value, not a variant rejected for reasons the event sender might not be aware of.

As for the naming - I don't see the problem with the `RequestTrackVariant + TrackVariantSwitched` pair. If that better describes the intent of the event, then I'd prefer it over the "symmetricity" of the names